chsdi/templates/htmlpopup/unesco_bak.mako

```diff
@@ -4,9 +4,25 @@
 
 <%def name="table_body(c, lang)">
 <%
-    surface_ha = int(float(c['attributes']['bgdi_surface']) / 10000) 
+    surface_ha = int(float(c['attributes']['bgdi_surface']))
+    if lang == 'fr': 
+        name = 'name_fr'
+        type = 'type_fr'
```

Comment:
`type` is a reserved word in python (that's why it appears blue). Can you use another variable name?